hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -1695,6 +1695,15 @@
     </description>
   </property>
 
+  <property>
+    <name>hdds.scm.safemode.reported.datanode.pct</name>
+    <value>0.90</value>
```

Comment:
I think 90% is too much and a significant difference from previous config, If there is a cluster say with under-utilized DN's on which which there is no data in ~~30-40% of total DN's , Safemode would still wait for these to be registered. IMO DatanodeSafemodeRule is to ensure there are datanodes available for a write to go through. We already do the check to see if enough containers are available for reading in the containerSafemodeRule

cc @nandakumar131

Reply:
Thank you very much for helping review the code! From my personal perspective, I believe we should still have an optional configuration to control this. You made a valid point—`0.9` might be a relatively large value, but if only one DN is registered and the rule passes, it seems a bit too lenient. We set the default value to `0.1`. Do you think that would be acceptable?

Reply:
I think 0.1 sounds good, thanks

Reply:
I also think that the Datanode safe mode rule is meant to ensure writes work. So that means we only need one Datanode to be present as Ozone still allows single replica writes.